src/mbgl/util/geometry_util.hpp

```diff
@@ -20,34 +20,34 @@ const GeometryBBox<double> DefaultDistanceBBox = std::array<double, 4>{{std::num
                                                                         -std::numeric_limits<double>::infinity()}};
 
 template <typename T>
-void updateBBox(GeometryBBox<T>& bbox, const Point<T>& p);
+void updateBBox(GeometryBBox<T>& bbox, const Point<T>& p) noexcept;
```

Comment:
Not 100% sure if we can use `noexcept` on these (because of the template parameter).